gpkit/small_scripts.py

```diff
@@ -189,7 +189,8 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
     decorated = [(bool(v.shape) if isinstance(v, Iterable) else False,
                  (varfmt % k),
                  i, k, v) for i, (k, v) in enumerate(data.items())
-                 if np.max(abs(v)) >= minval]
+                 if (np.max(abs(v)) >= minval) | (np.any(np.isnan(v)))]
+                #                               allows nans to be printed 
```

Comment:
Hmm, this PR seems to include #298, so we should probably just close that one if we end up merging this.